core/src/db/snapshot/Snapshot.h

```diff
@@ -56,8 +56,7 @@ class Snapshot : public ReferenceProxy {
         return it->first;
     }
 
-    CollectionPtr
-    GetCollection() {
+    CollectionPtr GetCollection() {
```

Comment:
your clang-format should update to clang-format-7

Reply:
Ok